hw/top_earlgrey/ip/ast/doc/_index.md

```diff
@@ -0,0 +1,1006 @@
+</tr>
+<tr class="even">
+<td>ts_alert_hi_trig_i</td>
+<td>I</td>
+<td>1</td>
+<td>alert</td>
+<td>Alert force trigger by software - connected to a chip register</td>
+</tr>
+<tr class="odd">
+<td>ts_alert_lo_po/no</td>
+<td>O</td>
+<td>2</td>
+<td>alert</td>
+<td>Temperature sensor alert for low temperature band</td>
+</tr>
+<tr class="even">
+<td>ts_alert_lo_ack_i</td>
+<td>I</td>
+<td>1</td>
+<td>alert</td>
+<td>single pulse ack, source and destination assumed to be synchronous</td>
+</tr>
+<tr class="odd">
+<td>ts_alert_lo_trig_i</td>
+<td>I</td>
+<td>1</td>
+<td>alert</td>
+<td>Alert force trigger by software - connected to a chip register</td>
+</tr>
+<tr class="even">
+<td>ls_alert_po/no</td>
+<td>O</td>
+<td>2</td>
+<td>alert</td>
+<td>Light sensor alert (TBD)</td>
+</tr>
+<tr class="odd">
+<td>ls_alert_ack_i</td>
+<td>I</td>
+<td>1</td>
+<td>alert</td>
+<td>single pulse ack, source and destination assumed to be synchronous</td>
+</tr>
+<tr class="even">
+<td>ls_alert_trig_i</td>
+<td>I</td>
+<td>1</td>
+<td>alert</td>
+<td>Alert force trigger by software - connected to a chip register</td>
+</tr>
+<tr class="odd">
+<td>ot_alert_po/no</td>
+<td>O</td>
+<td>TBD</td>
+<td>alert</td>
+<td>Other alerts (TBD)</td>
+</tr>
+<tr class="even">
+<td>ot_alert_ack_i</td>
+<td>I</td>
+<td>1</td>
+<td>alert</td>
+<td>single pulse ack, source and destination assumed to be synchronous</td>
+</tr>
+<tr class="odd">
+<td>ot_alert_trig_i</td>
+<td>I</td>
+<td>1</td>
+<td>alert</td>
+<td>Alert force trigger by software - connected to a chip register</td>
+</tr>
+<tr class="even">
+<td><strong>Trimming Test and Debug</strong></td>
+<td></td>
+<td></td>
+<td></td>
+<td></td>
+</tr>
+<tr class="odd">
+<td>scan_mode_i</td>
+<td>I</td>
+<td>1</td>
+<td></td>
+<td>Scan mode indication signal. Controllable only when DFT features are enabled (Test and RMA states). Otherwise, these signals are grounded to 0.</td>
+</tr>
+<tr class="even">
+<td>scan_reset_ni</td>
+<td>I</td>
+<td>1</td>
+<td></td>
+<td>Scan reset</td>
+</tr>
+<tr class="odd">
+<td>clk_ast_ext_i</td>
+<td>I</td>
+<td>1</td>
+<td>async</td>
+<td><p>External clock. While AST generates most of its clocks on-die, it still needs an external clock for clock calibration and first flash/OTP programming.</p>
+<p>Clock calibration: AST clock sources are inaccurate by default and must be calibrated prior to use. The results of the calibration are stored in <a href="https://github.com/lowRISC/opentitan/tree/master/hw/ip/otp_ctrl"><u>OTP</u></a> and reloaded by software upon system boot.</p>
+<p>First Flash / OTP programming: AST clock sources are inaccurate by default and may be out of range for initial flash and OTP programming. In this situation, an external clock may be required for initial programming such that a software image can be loaded to calibrate clocks and advance <a href="https://docs.opentitan.org/doc/security/"><u>life cycle</u></a>.</p></td>

```

Comment:
might be good to take out the life cycle link too, and we will add it when ready.